test-in-qmlscene.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+PS3='
+Test a watchface in qmlscene.
+Enter watchface number.
+Quit with any other input -> '
+
+unset options i
+while IFS= read -r -d $'\0' f; do
+  options[i++]="$f"
+done < <(find */ -maxdepth 0 -type d -print0 )
+
+if [ "$1" = "-use12h" ]
+then
+    use12=true
+else
+    use12=false
+fi
+
+if ! command -v qmlscene &> /dev/null
+then
+    echo "$(tput setaf 1)qmlscene could not be found. Install qtcreator from your package manager.$(tput sgr0)"
+    exit
+else
+    echo "$(tput setaf 2)qmlscene found, proceeding...$(tput sgr0)
+         "
+fi
+
+select opt in "${options[@]}"
+
+do
+    if [ -e $opt/usr/share/ ]
+    then
+          sed '/Item {/a height: 640; width: 640; Image { source: "background.jpg"; width: 640; height: 640;} property bool displayAmbient: false' <$opt/usr/share/asteroid-launcher/watchfaces/${opt::-1}.qml >$opt/usr/share/asteroid-launcher/watchfaces/${opt::-1}_scene.qml
```

Comment:
Just "Item {" is a bit fragile. I am not entirely sure about sed's syntax but maybe we can slightly improve the matching either by limiting to the first replace (maybe sed has an option for that, idk), or match the only "Item {" at the beginning of the line (in grep syntax that would be ^Item { I think